pkg/operator/controller/ingress/namespace.go

```diff
@@ -155,3 +157,17 @@ func (r *reconciler) ensureRouterClusterRoleBinding() error {
 	}
 	return nil
 }
+
+func (r *reconciler) ensureRouterNamespaceNetworkPolicy(ci *operatorv1.IngressController) (bool, *networkingv1.NetworkPolicy, error) {
+	np := manifests.RouterNetworkPolicyDenyAll()
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: np.Namespace, Name: np.Name}, np); err != nil {
+		if !errors.IsNotFound(err) {
+			return false, nil, fmt.Errorf("failed to get router network policy %s/%s: %v", np.Namespace, np.Name, err)
+		}
+		if err := r.client.Create(context.TODO(), np); err != nil {
+			return false, nil, fmt.Errorf("failed to create router network policy %s/%s: %v", np.Namespace, np.Name, err)
```

Comment:
This is the "default namespace network policy", not the "router network policy".  Anyway, `ensureIngressController` wraps these errors, so the wrapping here seems redundant.